lux/core/series.py

```diff
@@ -43,6 +43,7 @@ class LuxSeries(pd.Series):
         "_prev",
         "_history",
         "_saved_export",
+        "name",
```

Comment:
"name" is already added to the bottom of this list (will be added twice)

Reply:
Oh cool, good to know. Will remove this then